mscore/seq.cpp

```diff
@@ -46,6 +46,7 @@
 
 #include "click.h"
 
+#define OV_EXCLUDE_STATIC_CALLBACKS
```

Comment:
I have no idea (anymore) why I though this to be needed. Compiles without warning on MinGW 64bit and MSCV